libethdrivers/src/debug.h

```diff
@@ -80,7 +80,7 @@ static inline void print_val(uint8_t *p, int len)
 
 static inline void print_packet(const char *col, void *packet, int length)
 {
-    cprintf(col, "packet 0x%x (%d bytes)", (uint32_t)packet, length);
+    cprintf(col, "packet 0x%x (%d bytes)", (uintptr_t)packet, length);
```

Comment:
```suggestion
    cprintf(col, "packet 0x%"PRIxPTR" (%d bytes)", (uintptr_t)packet, length);
```
would be technically correct for `uintptr_t`. May not matter practically, though.